static/app/components/onboardingWizard/filterSupportedTasks.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  feedbackOnboardingPlatforms,
+  replayOnboardingPlatforms,
+  withoutPerformanceSupport,
+} from 'sentry/data/platformCategories';
+import {type OnboardingTask, OnboardingTaskKey, type Project} from 'sentry/types';
+
+const replayRelatedTasks = [OnboardingTaskKey.SESSION_REPLAY];
+const performanceRelatedTasks = [
+  OnboardingTaskKey.FIRST_TRANSACTION,
+  OnboardingTaskKey.PERFORMANCE_GUIDE,
+  OnboardingTaskKey.METRIC_ALERT,
+];
+const feedbackRelatedTasks = [OnboardingTaskKey.USER_REPORTS];
+
+export function filterSupportedTasks(
+  projects: Project[] | undefined,
+  allTasks: OnboardingTask[]
+): OnboardingTask[] {
+  if (!projects) {
+    return [];
+  }
+  // Remove tasks for features that are not supported
+  return allTasks.filter(
+    task =>
+      (shouldShowReplayTasks(projects) || !replayRelatedTasks.includes(task.task)) &&
```

Comment:
my boolean algebra is failing me. why is this `||` with `!replayRelatedTasks.include(task.task)`?

Reply:
No I think you're right and I made a mistake. I even have a test for this which is not failing Lol! Let me see what I did wrong.

Reply:
i think it should be just `&&` and the in between parentheses should be `||`?

Reply:
The algebra is easier when you do the negative statement. We want to filter `out` a task if `!shouldShowReplayTasks && replayRelatedTasks.include(task)`.

So we want to filter `in` if `shouldShowReplayTasks(projects) || !replayRelatedTasks.includes(task.task)`

Reply:
ah i see